cfgov/v1/jinja2/v1/_includes/molecules/image-and-text-25-75.html

```diff
@@ -0,0 +1,29 @@
+
+{# ==========================================================================
+
+   image_and_text_25_75.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an image and text 25/75 molecule.
+   See [GHE]/flapjack/Modules-V1/wiki/25-75-Image-&-Text
+
+   template: Template URL for content to include.
+
+   img_url: URL for image to display.
+
+   ========================================================================== #}
+
+{% macro render(template, img_url) %}
+{% import template as content %}
+<div class="m-image-and-text-25-75">
+    <div class="m-image-and-text-25-75_image-container">
+        <img src="{{ img_url }}">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5ac8bf0799c8b21db7fdb25ce75a515997daa7da